airbyte-integrations/connectors/source-freshsales/source_freshsales/schemas/completed_tasks.json

```diff
@@ -16,6 +17,9 @@
     "created_at": { "type": ["null", "string"] },
     "updated_at": { "type": ["null", "string"] },
     "outcome_id": { "type": ["null", "integer"] },
-    "task_type_id": { "type": ["null", "integer"] }
+    "task_type_id": { "type": ["null", "integer"] },
+    "updater_id": { "type": ["null", "integer"] },
+    "targetables": { "type": ["null", "string"] },
```

Comment:
I notice we dump a string that seems to always contain an array here, is that expected? E.g. for phone numbers and IDs, we output an array directly

Reply:
This is expected connectors behavior currently, not to break the other's connections. I Will dig deeper for GA cert.